pkg/server/manager/pipeline.go

```diff
@@ -112,60 +112,71 @@ func (m *pipelineManager) CreatePipeline(projectName string, pipeline *api.Pipel
 		return nil, err
 	}
 
+	gitSource, err := api.GetGitSource(pipeline.Build.Stages.CodeCheckout.MainRepo)
+	if err != nil {
+		return nil, err
+	}
+
 	// Create SCM webhook if enable SCM trigger.
-	var webHook *scm.WebHook
-	var gitSource *api.GitSource
-	if pipeline.AutoTrigger != nil && pipeline.AutoTrigger.SCMTrigger != nil {
-		gitSource, err = api.GetGitSource(pipeline.Build.Stages.CodeCheckout.MainRepo)
-		if err != nil {
-			return nil, err
+	createWebhook(pipeline, provider, scmConfig.Type, gitSource.Url, "")
```

Comment:
Check the returned error by `createWebhook()`?

Reply:
ok